content/changes.md

```diff
@@ -8,6 +8,32 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## February 29, 2024  
+
+### Improvements 
+
+- **Dart 3.3** 
+
+  Dart 3.3 is now supported in Qualified.
+
+### Challenges 
+
+- **Browser DevTools Quiz** 
+
+	This quiz tests a candidate's basic skill in working with the browser developer tools, an essential part of every developer's toolkit. 
```

Comment:
Looks like the indentation is off (I forgot to check the [preview](https://deploy-preview-204--fervent-mayer-2fb2ca.netlify.app/changes/#challenges) on the last PR):
![image](https://github.com/andela-technology/qualified-docs/assets/17895165/fae7b6cd-e071-489f-a17c-36ee4bb70947)
```suggestion
  This quiz tests a candidate's basic skill in working with the browser developer tools, an essential part of every developer's toolkit. 
```

I think the issue is tabs. Only use spaces.